docs/theme/components/collocator/index.tsx

```
@@ -0,0 +1,129 @@
+    attachedType,
+    functions,
+    codeFile,
+    componentInfo,
+    initValue,
+    messages,
+    setRadioValue,
+    setClearSign,
+    setConfig,
+  } = useCollocator({ api, name });
+
+  const renderHeader = (children: String | React.ReactElement, className?: string) => (
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)